client/my-sites/stats/stats-email-open-detail/index.jsx

```diff
@@ -300,7 +300,7 @@ class StatsEmailOpenDetail extends Component {
 									statType="opens"
 									postId={ postId }
 									siteId={ siteId }
-									period={ period }
+									period="alltime"
```

Comment:
I think this could be a constant. We put it in several places.

Reply:
You're right! I'll fix it.